### ISSUE TITLE
Error occurred into fake_sqs

### ISSUE THREAD

**Issue by kentokento** [evalphobia/dockerfiles#1]
```
/ # /usr/local/bin/go-sqs-create $SQS_QUEUES
2019/04/12 05:26:03 create queue name:my_queue1 ...
2019/04/12 05:26:03 [SQS error] name:my_queue1 error:SharedCredsLoad: failed to load shared credentials file
caused by: open /root/.aws/credentials: no such file or directory
```

I can avoid it below ... 😄 

```
  sqs:
    image: evalphobia/fake_sqs:1.0.0
    ports:
      - "14568:4568"
    environment:
      AWS_ACCESS_KEY_ID: "aws_access_key_id"
      AWS_SECRET_ACCESS_KEY: "aws_secret_access_key"
      AWS_SQS_ENDPOINT: "http://localhost:4568"
      SQS_QUEUES: "my_queue1 my_queue2 my_queue3"
```

**Comment by kentokento** [comment]
https://github.com/evalphobia/go-sqs-create/blob/master/config.go#L28-L38

**Comment by evalphobia** [comment]
Thanks.
It's fixed  in v1.0.2 https://github.com/evalphobia/go-sqs-create/pull/1
